src/yaml/paeng/29423-paeng-and-terring-amazed.yaml

```diff
@@ -0,0 +1,74 @@
+group: paeng
+name: paeng-and-terring-amazed
```

Comment:
Perhaps rename this simply to "amazed" and then add Terring to the authors. I would also make sure to rename the file and remove the `paeng-and-terring-` part from it.

Reply:
It's not a hard requirement as the files are actually identified by the Simtropolis id (for example when an author pushes an update), but the filename is generated based on the name, so I tend to try to keep them in sync.